dandiapi/api/views/auth.py

```diff
@@ -135,6 +135,7 @@ def user_questionnaire_form_view(request: AuthenticatedRequest) -> HttpResponse:
                     '@janelia.hhmi.org',
                     '@ccf.org',
                     '.ac.uk',
+                    '@mail.mcgill.ca',
```

Comment:
This doesn't seem to be universally the email domain for McGill folks--shouldn't this be simply `.mcgill.ca`?